### DISCUSSION TITLE
Used Space Not Shown

### DISCUSSION THREAD

**Discussion by chrisparana** [Netatalk/netatalk#702] [Q&A]
I'm having an issue where the used space on a share is not correct and always shows no space is being used. I set volume sizes for each of my shares. For example if I make a share 4GB, even after copying 2 or 3GB worth over to it, it still says 4GB are available. I had this happen on a previous install, and not sure what caused it, but some point it eventually worked correctly. I'm setting this up on a new server, and the issue has returned.

**Comment by rdmark** [comment]
@chrisparana Can you please provide some additional context:

- Version of netatalk
- OS version of client (new Macs or old Macs?)
- current configuration files
